shopfloor/actions/operation.py

```diff
@@ -0,0 +1,49 @@
+from odoo.tools.float_utils import float_compare
+
+from odoo.addons.component.core import Component
+
+
+class OperationAction(Component):
```

Comment:
I'm not fond of "operation" which doesn't really mean anything (or mean too many things in fact).

Maybe these could be on models though? `split_othen_move_lines` on StockMove and `planned_qty_in_location_is_empty` on StockLocation?

Reply:
Updated, better like this indeed, it makes the code clear about the intent (especially the `move.split_other_move_lines(move_line)`).